streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -220,10 +213,85 @@ public void tearDown() {
             streamsConfigUtils.close();
         if (adminClient != null)
             adminClient.close();
+        reset(streamThreadOne, streamThreadTwo);
+    }
+
+    private void prepareStreamsWithoutThread() {
```

Comment:
@FrankYang0529 Could you please separate this class to multi classes if those test cases require different setup?